tests/test_hep_bd1xx.py

```diff
@@ -168,18 +168,18 @@ def test_authors_from_100__a_e_w_y_and_700_a_e_w_y():
 
     snippet = (
         '<record>'
-        '  <datafield tag="100" ind1=" " ind2=" ">'
-        '    <subfield code="a">Vinokurov, Nikolay A.</subfield>'
-        '    <subfield code="e">ed.</subfield>'
-        '    <subfield code="w">N.A.Vinokurov.2</subfield>'
-        '    <subfield code="y">0</subfield>'
-        '  </datafield>'
         '  <datafield tag="700" ind1=" " ind2=" ">'
         '    <subfield code="a">Knyazev, Boris A.</subfield>'
         '    <subfield code="e">ed.</subfield>'
         '    <subfield code="w">B.A.Knyazev.2</subfield>'
         '    <subfield code="y">0</subfield>'
         '  </datafield>'
+        '  <datafield tag="100" ind1=" " ind2=" ">'
+        '    <subfield code="a">Vinokurov, Nikolay A.</subfield>'
+        '    <subfield code="e">ed.</subfield>'
+        '    <subfield code="w">N.A.Vinokurov.2</subfield>'
+        '    <subfield code="y">0</subfield>'
+        '  </datafield>'
```

Comment:
Can't you add another test, instead of modifying a previous one? Just so that we can give it a significant test name, like
`test_authors_from_700__a_and_100__a_preserves_ordering_of_authors`.